src/components/ManageLoginData.jsx

```diff
@@ -0,0 +1,11 @@
+import React, {useState} from 'react';
+
+import Login from './Login';
+
+const ManageLoginData = () => {
+
+    return (
+        <Login/>
+    );
+};
+export default ManageLoginData;
```

Comment:
this can be clean up

```suggestion
const ManageLoginData = () => (
     <Login/>
);

export default ManageLoginData;
```